patterns/create_mnemonic_phrases/disclaimer.md

```diff
@@ -0,0 +1,4 @@
+# DISCLAIMER
```

Comment:
Do we really need this disclaimer? All of fabric is under the MIT license

Reply:
You're right - the MIT license already contains a disclaimer. I was just a bit overcautious.
I updated the pull request.